src/components/Notification/Notification.js

```diff
@@ -158,16 +151,14 @@ export class ToastNotification extends Component {
   useIcon = kindProp =>
     ({
       error: iconErrorSolid,
-      info: iconInfoSolid,
       success: iconCheckmarkSolid,
-      warning: iconWarningSolid,
+      warning: iconErrorSolid,
```

Comment:
Just wanted to double-check if the icon list change here is intentional.

Reply:
Thanks for catching that, my mistake! This branch is still WIP so it just contained work I wanted to temporarily save and check into git